packages/now-cli/test/dev-server.unit.js

```diff
@@ -399,13 +414,15 @@ test('[DevServer] parseListen()', t => {
   t.deepEqual(parseListen('3000'), [3000]);
   t.deepEqual(parseListen('0.0.0.0'), [3000, '0.0.0.0']);
   t.deepEqual(parseListen('127.0.0.1:3005'), [3005, '127.0.0.1']);
-  t.deepEqual(parseListen('tcp://127.0.0.1:5000'), [5000, '127.0.0.1']);
-  t.deepEqual(parseListen('unix:/home/user/server.sock'), [
-    '/home/user/server.sock',
-  ]);
-  t.deepEqual(parseListen('pipe:\\\\.\\pipe\\PipeName'), [
-    '\\\\.\\pipe\\PipeName',
-  ]);
+  if (process.platform !== 'win32') {
+    t.deepEqual(parseListen('tcp://127.0.0.1:5000'), [5000, '127.0.0.1']);
```

Comment:
Weird that that one is failing 🤔

Reply:
I might have skipped this pre-maturely. You think its just the `tcp:` one or all of them?

Reply:
Just the `tcp` one.

Reply:
Ok, the test was added back for windows 👍